libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.m

```diff
@@ -162,6 +162,19 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task willPer
     }
 }
 
+- (BOOL)isSalesforceURL:(NSURL *)url {
+    // List from https://help.salesforce.com/s/articleView?language=en_US&id=sf.domain_name_url_formats.htm&type=5
```

Comment:
Is here any way to get this list via API so we don't have to manually keep track of changes?

Reply:
I haven't found anything :/